README.rst

```diff
@@ -8,7 +8,7 @@ Running Howitz step-by-step overview
 
 1. Preparation step:
     1. Make sure you have ``python3``, ``pip`` and ``git`` installed.
-    2. Download `Howitz from the GitHub repo <https://github.com/Uninett/Howitz>`_.
+    2. Download Howitz from the `GitHub repo <https://github.com/Uninett/Howitz>`_.
```

Comment:
I think the first version of the link text was more descriptive (i.e. it contains the keyword `Howitz`, which is good for the search engines, you know)